Sources/Transaction/Memos/RecoverableMemo.swift

```diff
@@ -17,22 +16,31 @@ enum RecoverableMemo {
     case sender(RecoverableSenderMemo)
     case destination(RecoverableDestinationMemo)
     case senderWithPaymentRequest(RecoverableSenderWithPaymentRequestMemo)
-    
+
     init(decryptedMemo data: Data66, accountKey: AccountKey, txOutKeys: TxOut.Keys) {
         guard let memoData = Data64(data[2...]) else {
             logger.fatalError("Should never be reached because the input data > 2 bytes")
         }
         let typeBytes = data[..<2]
-        
+
         switch typeBytes.hexEncodedString() {
         case Types.SENDER_WITH_PAYMENT_REQUEST:
-            let memo = RecoverableSenderWithPaymentRequestMemo(memoData, accountKey: accountKey, txOutPublicKey: txOutKeys.publicKey)
+            let memo = RecoverableSenderWithPaymentRequestMemo(
+                memoData,
+                accountKey: accountKey,
+                txOutPublicKey: txOutKeys.publicKey)
             self = .senderWithPaymentRequest(memo)
         case Types.SENDER:
-            let memo = RecoverableSenderMemo(memoData, accountKey: accountKey, txOutPublicKey: txOutKeys.publicKey)
+            let memo = RecoverableSenderMemo(
+                memoData,
+                accountKey: accountKey,
+                txOutPublicKey: txOutKeys.publicKey)
```

Comment:
double-indent function call params:
```suggestion
                    memoData,
                    accountKey: accountKey,
                    txOutPublicKey: txOutKeys.publicKey)
```

Reply:
single indentation is the general rule